workbench-core/authorization/src/dynamicAuthorization/ddbDynamicAuthorizationPermissionsPlugin.test.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  DynamoDBClient,
+  ServiceInputTypes,
+  ServiceOutputTypes,
+  TransactWriteItemsCommand,
+  TransactionCanceledException
+} from '@aws-sdk/client-dynamodb';
+import { AwsService } from '@aws/workbench-core-base';
+import { mockClient, AwsStub } from 'aws-sdk-client-mock';
+import { Action } from '../action';
+import { IdentityPermissionCreationError } from '../errors/identityPermissionCreationError';
+import { ThroughputExceededError } from '../errors/throughputExceededError';
+import { Effect } from '../permission';
+import { DDBDynamicAuthorizationPermissionsPlugin } from './ddbDynamicAuthorizationPermissionsPlugin';
+import { IdentityPermission, IdentityType } from './dynamicAuthorizationInputs/identityPermission';
+
+describe('DDB Dynamic Authorization Permissions Plugin tests', () => {
+  const region = 'us-east-1';
```

Comment:
Jest highly recommends that initializations happen in a `beforeEach/All` block because of how it loads tests and allows customization.

Reply:
is there documentation on this?